LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I decided to create this LICENSE file so that we have directly an icon for the license on github